src/main/java/reposense/authorship/FileInfoExtractor.java

```diff
@@ -128,11 +127,10 @@
      * Returns a {@code Set} of non-binary files for the repo {@code repoConfig}.
      */
     public static Set<Path> getListOfNonBinaryFiles(RepoConfiguration repoConfig) {
```

Comment:
just to standardize the naming
`getListOfNonBinaryFiles` -> `getNonBinaryFileList`